app/scripts/lib/transaction/metrics.ts

```diff
@@ -978,6 +979,13 @@ async function buildEventFragmentProperties({
     uiCustomizations.push(MetaMetricsEventUiCustomization.GasEstimationFailed);
   }
 
+  if (
+    REDESIGN_TRANSACTION_TYPES.includes(transactionMeta.type as TransactionType)
```

Comment:
Following the developer settings PR, this will be static and not based on the ENV, so do we also want to temporarily check the env / preference here? @pnarayanaswamy

Depends which PR goes in first.

Reply:
added in 50e55166aa